Libraries/LibWeb/HTML/PopoverInvokerElement.idl

```diff
@@ -0,0 +1,11 @@
+[MissingValueDefault=toggle, InvalidValueDefault=toggle]
```

Comment:
Could we add a spec link here: https://html.spec.whatwg.org/multipage/popover.html#attr-popovertargetaction